irrep/bandstructure.py

```diff
@@ -1314,6 +1329,11 @@ def write_trace_all(
         """
         f = open(fname, "w")
         kpline = self.KPOINTSline()
+        json_data = {}
```

Comment:
Is this json object used somewhere? It is not returned by the function, neither saved before the function reaches the end.

Reply:
Yes, this one should be removed